External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -177,6 +179,22 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 
 newNifti.dat(:,:,:,:,:) = imNew;
 
+
+%% 4. Remove redundant .mat orientation files
+if size(imNew,4)==1
+    xASL_delete(tempMat);
+end
+
+if exist(tempMat, 'file')
+	tmpMat = load(tempMat);

```

Comment:
It is all fine. But we might want to rename `tmpMat` to something else as it looks a bit like a typo in `tempMat`.

Reply:
I'm probably not the one who wrote this code, knowing how I dislike such abbreviations ;)
Renamed to motionMat